### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by vladislav-prishchepa**

### CODE REVIEW

**Comment by vtyrin-pyrus** [comment]
@vladislav-prishchepa will take a look later today

File: src/ITCC.HTTP.SslConfigUtil.Console/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,11 +8,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("ITCC.HTTP.SslConfigUtil.Console")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("Comand line tool for binding SSL certificates to HttpListener. Self-signed certificate generation included.")]
```
Comment (VladimirTyrin) [inline src/ITCC.HTTP.SslConfigUtil.Console/Properties/AssemblyInfo.cs]: for HttpListener

File: src/ITCC.HTTP.SslConfigUtil.Core/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,11 +8,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("ITCC.HTTP.SslConfigUtil.Core")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("Class library for binding SSL certificates to HttpListener. Self-signed certificate generation included.")]
```
Comment (VladimirTyrin) [inline src/ITCC.HTTP.SslConfigUtil.Core/Properties/AssemblyInfo.cs]: for HttpListener SSL certificate binding

File: src/ITCC.HTTP.SslConfigUtil.GUI/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -10,11 +10,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("ITCC.HTTP.SslConfigUtil.GUI")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("GUI tool for binding SSL certificates to HttpListener. Self-signed certificate generation included.")]
```
Comment (VladimirTyrin) [inline src/ITCC.HTTP.SslConfigUtil.GUI/Properties/AssemblyInfo.cs]: for HttpListener SSL certificate binding

File: src/ITCC.Logging.Reader.Core/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -8,11 +8,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("ITCC.Logging.Reader.Core")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("Base class library for parsing file logs")]
```
Comment (VladimirTyrin) [inline src/ITCC.Logging.Reader.Core/Properties/AssemblyInfo.cs]: for ITCC.Logging.Logger log files parsing

**Pull request merged**